### ISSUE TITLE
Project: Docker based system to assess challenge submissions

### ISSUE THREAD

**Issue by royak** [NA-MIC/ProjectWeek#735]
### Project 

Designing a Docker-based system to assess the submissions of challenge participants.

### Category

Infrastructure

### Presenter Location

Online

### Key Investigators

- Roya Khajavibajestani (Brigham and women’s hospital, USA)
- Ron Kikinis (Harvard Medical School, USA)
- Steve Pieper( Isomics, USA)

### Project Description

Project Description:

Our project is focused on developing a Docker-based submission mechanism for challenge participant. To maintain fairness and make sure that the test set is not used in the training process, the test set will not be released to the participants. Instead, participants will be required to containerize their methods using Docker and submit their Docker containers for evaluation. 

Docker provides an excellent solution for running algorithms in isolated environments known as containers. In our project, we will leverage Docker to create a container that replicates the participants' pipeline requirements and executes their inference script. By encapsulating the entire environment within a container, we can ensure consistent execution and reproducibility.

### Objective

Create a sample docker container for submission
Create an evaluation mechanism on the challenge website
Create documentation, guidelines, and tutorial for participants

### Approach and Plan

Design the docker container, input/output mechanism, requirements, and methods to perform inference using a subset of the validation set.
Create an evaluation mechanism on the challenge website
Create a sample submission docker for the test phase and test it on the challenge website
Create documentation to publish in phase 2 of the challenge.

### Progress and Next Steps

1. Describe specific steps you **have actually done**.
